exercises/exercise4.js

```diff
@@ -0,0 +1,32 @@
+//  function that checks the level of nesting of an array
+function createSingleDimensionArray() {
```

Comment:
On this task, we were asked to create a function helper to calculate the level of nesting of an array and then use that helper function to flatten an array.